src/like.js

```diff
@@ -0,0 +1,67 @@
+const apiUrl = "https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/";
+let likes = {};
+
+document.addEventListener("DOMContentLoaded", () => {
+    const likeButtons = document.querySelectorAll(".likeButton");
+    const likesCounts = document.querySelectorAll(".likesCount");
+
+    likeButtons.forEach((likeButton, index) => {
+        const item_id = likeButton.getAttribute("data-item");
+        likes[item_id] = localStorage.getItem(item_id) ? parseInt(localStorage.getItem(item_id)) : 0;
+
+        likeButton.addEventListener("click", async () => {
+            likes[item_id]++;
+            updateLikes(item_id);
+            await updateLikesOnAPI(item_id);
+            localStorage.setItem(item_id, likes[item_id]);
+        });
+
+        updateLikes(item_id);
+    });
+
+    fetchLikesFromAPI();
+});
```

Comment:
This all looks good and makes sense but it is not being reflected on the display of the code. Whenever I click a like button, nothing is shown to change. Please be careful and remember to add your event listeners!